experiments/jz/entity/exp_1/toy_example/03_create_dataset.slurm

```diff
@@ -47,10 +47,9 @@ REPO_DIR="${DATASETS_CUSTOM}/${myarray[-1]}"
 python bsmetadata/train.py \
   data_config.experiment="with_metadata" \
   data_config.metadata_config.metadata_list=["entity"] \
-  data_config.metadata_config.max_seq_len=1024 \
```

Comment:
Out of curiosity, why did you remove this line? :blush:

Reply:
It's really as you wish for your experiences (I don't have a preference)! It's just that the gpt-2 model supports up to 1024 and wanted to make sure it wasn't from "bad" information that was given by the default settings.

Reply:
I didn't face any such issues as of now. Since, gpt-2 supports upto 1024, I'd prefer to use the same as well.